### DISCUSSION TITLE
Slow Consumer Reporter

### DISCUSSION THREAD

**Discussion by robinoffzewood** [nats-io/nats.rs#264] [Q&A] [answered]
Is there a way to detect that the client is a slow consumer?
For example for some clients there are possiblities described here: https://docs.nats.io/developing-with-nats/events/slow

**Comment by Jarema** [comment, marked as answer]
Hey.

Sorry for the late answer.

#296 will have a follow-up with PR shortly.

**Comment by robinoffzewood** [comment]
thanks!
